cpplite/include/http/libcurl_http_client.h

```diff
@@ -268,6 +269,45 @@ namespace azure {  namespace storage_lite {
             return actual_size;
         }
 
+        static int debug_callback(CURL *handle, curl_infotype type, char *data, size_t size, void *userptr)
```

Comment:
Can you post some sample of how this log looks.

Reply:
Aug  3 09:47:14 gapra-Virtual-Machine blobfuse[177172]: => Send header: GET /fusetmp?comp=list&delimiter=/&include=metadata&maxresults=1&restype=container HTTP/1.1#015#012Host: gaprauswest2.blob.core.windows.net#015#012Accept: */*#015#012User-Agent: azure-storage-fuse/1.4.5-preview#015#012x-ms-date: Wed, 03 Aug 2022 16:47:14 GMT#015#012x-ms-version: 2018-11-09#015#012Authorization: SharedKey gaprauswest2:V+pmBXqZtwaL6+fxJfvQcGYBCX9+iJ4BnLm0WmSyCGQ=#015#012#015
Aug  3 09:47:14 gapra-Virtual-Machine blobfuse[177172]: == Info: Mark bundle as not supporting multiuse
Aug  3 09:47:14 gapra-Virtual-Machine blobfuse[177172]: <= Recv header: HTTP/1.1 200 OK#015
Aug  3 09:47:14 gapra-Virtual-Machine blobfuse[177172]: <= Recv header: Transfer-Encoding: chunked#015
Aug  3 09:47:14 gapra-Virtual-Machine blobfuse[177172]: <= Recv header: Content-Type: application/xml#015

Reply:
can we remove those "<=" and just add may be "libcurl:" so that we know this is coming from libcurl and not our usual logs. Just that it helps to filter out libcurl logs for debugging later.